Sources/ProjectSpec/Dependency.swift

```diff
@@ -31,6 +33,17 @@ public struct Dependency: Equatable {
         self.link = link
         self.implicit = implicit
         self.weakLink = weakLink
+        self.platform = platform
+    }
+    
+    public enum Platform: Equatable {
+        case all
+        case iOS
+        case mac
+        
+        public static func == (lhs: Self, rhs: Self) -> Bool {
```

Comment:
This equatable function isn't required, and in this case actually returns the wrong result

Reply:
If you make Platform a RawRepresentable `Platform: String, Equatable` Swift can auto generate the Equatable conformance. This also makes the decoding code simpler as you can init with a string

Reply:
I refactored dependence equatable in a1ebf2240158f49f99a71ed3cac5a4a5c60308e4.